tests/unit/stores/pacbioLibraries.spec.js

```diff
@@ -12,80 +34,49 @@ describe('usePacbioLibrariesStore', () => {
     setActivePinia(pinia)
   })
 
-  describe('state', () => {
-    let libraries
-    beforeEach(() => {
-      libraries = new Response(Data.TractionPacbioLibrary).deserialize.libraries
+  describe('#libraryPayload', () => {
```

Comment:
Wondering if this should be moved to a separate utilities test.

Reply:
I did think about that but because it was only one function not sure it is worth it. If the utilities file expands then definitely. I wanted to put this function in the store but it doesn't make it easy to test.